File: python/empleado.py
"""diccionario de datos de empleados"""
from utilidades  import *
from typing import Dict, Any

def main():
    #definir diccionario
    empleado: Dict[str, Any] = {
        'ci': '30640501',
        'nombre': 'Juan Carlos',
        'edad': 25,
        'cargo': 'Desarrollador',
        'salario': '5000',
    }
    encabezado('datos del empleado', 50)
    for clave, valor in empleado.items():
        if isinstance(valor, int ):
            print(f"{clave.upper()}: {valor} años")
        else: 
            print(f"{clave.upper()}: {valor}")
    linea(50)
        
main()

File: JS/src/empleado.js
/* datos del empleado */
import * as util  from "./utilidades.js";

const main = () => {
//declara variables crear un objetivo de js
const empleado = {
    'cedula': '30640501',
    'nombre': 'dylan',
    'apellido': 'Sosa',
    'edad': 20,
    'sueldo': 20000
}

util.encabezado('datos del empleado', 50);
for (const clave in empleado) {
   console.log(`${clave.toUpperCase()}: ${empleado[clave]}`)
        
    }
    //conveertir objeto a un json (texto plano)
    util.linea(50)
    const empleadoJSON = JSON.stringify(empleado);
    console.log(`JSON: empleadoJSON`);

    //convertir un json a un objeto js
    util.linea(50)
    const empleadoObject = JSON.parse(empleadoJSON)
    for (const clave in empleadoObject) {
        console.log(`${clave.toUpperCase()}: ${empleadoObject[clave]}`)
            
        }
}


main()